snippets/base/storage.py

```diff
@@ -7,3 +15,39 @@ def get_available_name(self, name):
         if self.exists(name):
             self.delete(name)
         return name
+
+
+@deconstructible
+class S3Storage(S3BotoStorage):
+    cache_control_headers = getattr(settings, 'AWS_CACHE_CONTROL_HEADERS', {})
+
+    def _save(self, name, content):
+        cleaned_name = self._clean_name(name)
+        name = self._normalize_name(cleaned_name)
+        headers = self.headers.copy()
+        for filename_start, value in self.cache_control_headers.iteritems():
+            if name.startswith(filename_start):
+                headers['Cache-Control'] = value
```

Comment:
The Cache-Control header appears to be the only difference from the current https://github.com/jschneier/django-storages/blob/master/storages/backends/s3boto.py -- it seems like a reasonable change, but is there (or should there be) a PR against the main repo to incorporate this upstream?